src/helpers/search.js

```diff
@@ -60,7 +63,25 @@ const fuse = new Fuse([], fuseOptions)
 export const filterContactsBySearch = (contacts, searchValue) => {
   if (searchValue.length === 0) return contacts
 
+  if (flag('search-threshold')) {
+    fuse.options.threshold = window.threshold
+    console.info('fuse threshold :', fuse.options.threshold) //eslint-disable-line no-console
+  }
+
   fuse.setCollection(contacts)
   const fuseResults = fuse.search(searchValue)
   return fuseResults.map(result => result.item)
 }
+
+const setThreshold = value => {
+  if (!isNaN(value)) {
+    console.info('changing fuse threshold to', value) //eslint-disable-line no-console
+    console.info('please change the search value') //eslint-disable-line no-console
+    window.threshold = value
```

Comment:
No need to store it in `window`. Just set the flag value and read from the flag.